website/docs/r/storage_table.html.markdown

```diff
@@ -68,6 +68,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `id` - The ID of the Table within the Storage Account.
 
+* `resource_manager_id` - The ID of the Table in the Resource Manager format, e.g. "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Storage/storageAccounts/myAccount/tableServices/default/tables/myTable". Useful for cases like IAM that do not use the Data Plane ID.
```

Comment:
I was conflicted about mentioning a use-case, whether we should let the user decide on how they might use a feature.